kpops/component_handlers/kafka_connect/kafka_connect_handler.py

```diff
@@ -63,12 +63,15 @@ def create_connector(
                     secs=self._timeout,
                 )
 
-    def destroy_connector(self, connector_name: str, *, dry_run: bool) -> None:
+    def destroy_connector(
+        self, connector_config: KafkaConnectorConfig, *, dry_run: bool
```

Comment:
why do we need to pass the entire connector config for destroying if we only need the name?

Reply:
I wanted to be consistent. Similarly, we are passing the connector config to the `deploye_connector` method (because of logging reasons).

Reply:
do you mean `create_connector`?

Reply:
@disrupted Yes :D typo gefixed